src/test/cdp.test/cdp.test.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Rename "cdp.text.csproj" to "Tree.Test.csproj"

Reply:
Done